frontend/lib/resolution/Resolver.cpp

```diff
@@ -2504,18 +2514,10 @@ void Resolver::exit(const New* node) {
 
     // TODO: Need to also print the type name.
```

Comment:
This TODO is now implemented by `ErrorMemManagementNonClass::write` and can be removed